config/v1/types_console.go

```diff
@@ -24,9 +24,9 @@ type ConsoleSpec struct {
 }
 
 type ConsoleStatus struct {
-	// The hostname for the console. This will match the host for the route that
+	// The URL for the console. This will be derived from the host for the route that
 	// is created for the console.
-	PublicHostname string `json:"publicHostname"`
+	PublicURL string `json:"publicURL"`
```

Comment:
Maybe `consoleURL` in case we ever have another URL we publish in status?

Reply:
```yaml
apiVersion: config.openshift.io/v1
kind: Console
metadata:
  name: cluster
spec:
  authentication:
    logoutRedirect: ""
status:
  # default url for config for console could be obvious its for the console
  publicURL: ""
  # or make it explicit 
  consoleURL: ""
  # for when we add others later
  otherURL: ""
  yetAnotherURL: ""
```
I don't personally have a strong opinion.

Reply:
`consoleURL` it is, updating.